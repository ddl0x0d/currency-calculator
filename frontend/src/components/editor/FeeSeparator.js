import Col from 'react-bootstrap/Col';
import React from 'react';
import Row from 'react-bootstrap/Row';

export default function FeeSeparator() {
  return (
    <Row className="justify-content-center">
      <Col md={8} xl={4}>
        <hr style={{ borderTop: '3px solid rgba(0, 0, 0, 0.5)' }} />
      </Col>
    </Row>
  );
}
