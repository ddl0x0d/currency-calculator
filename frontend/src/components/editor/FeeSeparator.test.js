import FeeSeparator from './FeeSeparator';
import { mount } from 'enzyme';
import React from 'react';

describe('<FeeSeparator />', () => {
  it('renders correctly', () => {
    const wrapper = mount(<FeeSeparator />);
    expect(wrapper).toMatchSnapshot();
  });
});
