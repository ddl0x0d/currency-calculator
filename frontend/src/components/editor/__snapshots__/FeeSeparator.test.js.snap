// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FeeSeparator /> renders correctly 1`] = `
<FeeSeparator>
  <Bootstrap(Row)
    className="justify-content-center"
  >
    <Row
      as="div"
      bsPrefix="row"
      className="justify-content-center"
      noGutters={false}
    >
      <div
        className="justify-content-center row"
      >
        <Col
          as="div"
          md={8}
          xl={4}
        >
          <div
            className="col-xl-4 col-md-8"
          >
            <hr
              style={
                Object {
                  "borderTop": "3px solid rgba(0, 0, 0, 0.5)",
                }
              }
            />
          </div>
        </Col>
      </div>
    </Row>
  </Bootstrap(Row)>
</FeeSeparator>
`;
